### DISCUSSION TITLE
Product Hunt Launch

### DISCUSSION THREAD

**Discussion by jekuer** [add2cal/add-to-calendar-button#440] [Announcements]
Hey GitHub community,

Today, we are live at Product Hunt!
Every Upvote counts. So, if you like this project and want to support its future development, please take the 5 minutes to hit the upvote button. Thanks a lot in advance! 🚀 

https://www.producthunt.com/posts/add-to-calendar-button-v2

Your Add to Calendar team! 😃

**Comment by jrod95** [comment]
Congratulation Jens, this is very exciting!
